app.js

```diff
@@ -39,7 +39,7 @@ app.use((req, res, next) => {
   res.header('Access-Control-Allow-Origin', '*');
   res.header(
     'Access-Control-Allow-Headers',
-    'Origin, X-Requested-With, Content-Type, Access, Authorization',
+    'Origin, X-Requested-With, Content-Type, Access, Authorization, x-access-token',
   );
```

Comment:
I needed to add this to include the token in the request header - hopefully that is ok.